pkg/cluster/describe.go

```diff
@@ -157,6 +157,20 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
 		isExistingVPC = "true"
 	}
 
+	// Check Hypershift-related values
+	mgmtCluster := ""
+	if cluster.Hypershift().Enabled() {
+		hypershiftResp, err := connection.ClustersMgmt().V1().Clusters().
+			Cluster(cluster.ID()).
+			Hypershift().
+			Get().
+			Send()
+		if err != nil {
+			return fmt.Errorf("unable to query hypershift endpoint for cluster %s: %v", cluster.ID(), err)
```

Comment:
This error should be silently ignored as there are only a handful of roles that can access this endpoint.